src/Horarium.InMemory/InMemoryRepository.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Horarium.Repository;
+
+namespace Horarium.InMemory
```

Comment:
This class is perfect for unit testing, sorry 😭

Reply:
:pepehands:

Some things... you know, they just must be done - i'll cover it with tests at some point - again, when i'll have the time